host/host.cpp

```diff
@@ -1,33 +1,77 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+#include <assert.h>
```

Comment:
I think in cpp you can `#include <cassert>`

https://en.cppreference.com/w/cpp/error/assert